README.md

```diff
@@ -84,12 +92,16 @@ bundle exec pipefymessage -w ConsumerExampleClass -R
 
 ### Development - Test
 
-To test changes without installing this dependency on your application, on your terminal go to the project root and execute:
+Run `make help` to see a list of all Make targets to help in common development
```

Comment:
What about changing this section instructions from a local env to the docker env that you added?

I mean, probably others will face the same problem that [you mentioned](https://github.com/pipefy/pipefy_message/pull/29/files#r911132957)...

WDYT?